docs/gettingstarted/proxies.md

```diff
@@ -0,0 +1,129 @@
+Using SidedProxies
+===================
+
+This page will go through all the benefits of using proxies and in what circumstance to use them in.
+
+What are sided proxies
+----------------------
+Sided proxies are a way to introduce side-specific things into your mod's code base in such a way that the other side will not crash when referencing classes intended for the other side. As a quick recap, sides here refer to either client or dedicated servers.
+
+An example of this would be a piece of code meant to register item models for rendering:
+
+```
+ModelLoader.setCustomModelResourceLocation(yourItem, 0, new ModelResourceLocation("modid:model_name", "inventory"));
+```
+
+While in your `@Mod` file you may want to just check for an appropriate side in one of the lifecycle events, it still won't help as the will still run in the server-side, where ModelResourceLocation cannot be found, causing an error.
```

Comment:
"it still won't help as the will still run in the server-side" might want to check your grammar there, might I suggest "it still won't help as it will still run on the server-side" instead.